### ISSUE TITLE
-g doesn't work for -i sat

### ISSUE THREAD

**Issue by karaluh** [Napsty/check_smart#55]
This is a regression, it used to work in 5.10. In the recent version I no longer see "sat" in the list of supported interfaces but the check, however, doesn't report any error:
```bash
sudo /usr/lib/nagios/plugins/check_smart.pl -g /dev/sd[a-b] -i sat
OK: [/dev/sda] - Device is clean|
```
There's also sdb though:
```bash
sudo /usr/lib/nagios/plugins/check_smart.pl -d /dev/sdb -i sat
OK: Drive  ST2000VN004-2E4164[...]
```
changing "sat" to either "ata" or "auto" leads to the same result: only the status of the first drive is reported.

**Comment by karaluh** [comment]
OK, I've found what was the problem, the device globbing must be put in quotation marks:
-g "/dev/sd[a,b]"
